include/sycl_ext_complex.hpp

```diff
@@ -1567,10 +1588,16 @@ operator<<(basic_ostream<_CharT, _Traits>& __os, const complex<_Tp>& __x)
 }
 #endif // !_SYCL_EXT_CPLX_HAS_NO_LOCALIZATION
 
+template<class _Tp, class = std::enable_if<is_gencomplex<_Tp>::value>>
+SYCL_EXTERNAL _SYCL_EXT_CPLX_INLINE_VISIBILITY
+inline const stream &operator<<(const stream &Out, const complex<_Tp>& RHS) {
```

Comment:
For clarity: 

```suggestion
inline const sycl::stream &operator<<(const sycl::stream &Out, const complex<_Tp>& RHS) {
```

(we should do the same in the comment)

Reply:
Alors for the naming, should we stay coherent with the `<<(basic_ostream` and use
`const stream &__ss, const complex<_Tp>& __x` ?

Reply:
That is a great idea, keeps things consistent. I have made the change.